packages/graphql-server/src/resolvers/appointment/api.ts

```diff
@@ -0,0 +1,115 @@
+import { fetcher, setQueryParams } from '@reapit/elements'
+import logger from '../../logger'
+import { ServerContext } from '../../app'
+import errors from '../../errors'
+import { API_VERSION } from '../../constants/api'
+import {
+  GetAppointmentsArgs,
+  GetAppointmentByIdArgs,
+  UpdateAppointmentArgs,
+  CreateAppointmentArgs,
+  GetAppointmentsResult,
+  Appointment,
+} from './appointments'
+
+export const URLS = {
+  appointments: '/appointments',
+}
+
+export const callGetAppointmentsAPI = async (args: GetAppointmentsArgs, context: ServerContext) => {
```

Comment:
Can you add return types to your functions please

Reply:
@willmcvay 
I have cast the response's type bellow. TypeScript type-check system is intelligently enough to type check this function 

![image](https://user-images.githubusercontent.com/36730355/75138298-74e3ac00-571c-11ea-9eb4-25d1cf345ca5.png)

Reply:
2 points, Type Casting is not the same as Type Safety - the correct return type for this function is GetAppointmentResult | undefined because there is no guarantee the downstream will be successful. By casting the type, you are telling the compiler "I guarantee this will return 'foo' type, which is not accurate. Second point, it's about readability - this is going to be a public endpoint and I want people to explicitly be able to read what a function returns.

Reply:
updated